apps/ngx-env-demo/package.json

```diff
@@ -1,54 +1,47 @@
 {
-  "name": "@ngx-env/demo",
-  "version": "16.1.1",
-  "private": true,
+  "name": "ngx-env-demo",
+  "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "cross-env NODE_ENV=development ng serve",
-    "prepare:build": "npx ng config projects.ngx-env-demo.architect.build.options.ngxEnv.prefix 'NGX_'",
-    "build": "ng build",
-    "prepare:test": "npx ng config projects.ngx-env-demo.architect.test.options.ngxEnv.prefix 'NGX_'",
-    "test": "cross-env NODE_ENV=test NGX_BRANCH_NAME=main ng test --watch=false",
-    "prepare:server": "npx ng config projects.ngx-env-demo.architect.server.options.ngxEnv.prefix 'NGX_'",
-    "dev:ssr": "ng run ngx-env-demo:serve-ssr",
-    "serve:ssr": "node dist/ngx-env-demo/server/main.js",
-    "build:ssr": "ng build && ng run ngx-env-demo:server",
-    "prerender": "ng run ngx-env-demo:prerender"
+    "watch": "ng build --watch --configuration development",
+    "start": "NODE_ENV=development ng serve",
+    "prepare:build": "npx ng config projects.ngx-env-demo.architect.build.options.ngxEnv.prefix 'NG_APP_'",
+    "build": "NODE_ENV=development ng build",
+    "prepare:test": "npx ng config projects.ngx-env-demo.architect.test.options.ngxEnv.prefix 'NG_APP_'",
+    "test": "NODE_ENV=test NG_APP__BRANCH_NAME=main ng test --watch=false"
   },
+  "private": true,
   "dependencies": {
-    "@angular/animations": "^16.0.0",
-    "@angular/common": "^16.0.0",
-    "@angular/compiler": "^16.0.0",
-    "@angular/core": "^16.0.0",
-    "@angular/forms": "^16.0.0",
-    "@angular/localize": "^16.0.0",
-    "@angular/platform-browser": "^16.0.0",
-    "@angular/platform-browser-dynamic": "^16.0.0",
-    "@angular/platform-server": "^16.0.0",
-    "@angular/router": "^16.0.0",
-    "@nguniversal/express-engine": "^16.0.0",
+    "@angular/animations": "^17.0.0",
+    "@angular/common": "^17.0.0",
+    "@angular/compiler": "^17.0.0",
+    "@angular/core": "^17.0.0",
+    "@angular/forms": "^17.0.0",
+    "@angular/platform-browser": "^17.0.0",
+    "@angular/platform-browser-dynamic": "^17.0.0",
+    "@angular/platform-server": "^17.0.0",
+    "@angular/router": "^17.0.0",
+    "@angular/ssr": "^17.0.0",
     "@ngx-env/core": "file:../../packages/core/dist",
-    "express": "^4.15.2",
-    "rxjs": "~7.6.0",
-    "tslib": "^2.4.0",
-    "zone.js": "~0.13.0"
+    "@ngx-env/builder": "file:../../packages/builder/dist",
```

Comment:
should be a dev dep and point to "file:../../packages/builder" (remove dist) as this is where the package.json file lives.